src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A immutable interface to display a Peron's details
```

Comment:
Yes, not only person's details. Will give a better comment for my class in future!(: